.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_call:
+  pull_request:
+  schedule:
+    - cron: '53 0 * * *' # Daily at 00:53 UTC
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
```

Comment:
common workflow
canonical/data-platform-workflows/.github/workflows/lint.yaml@v0.0.0

Reply:
Added in https://github.com/canonical/data-platform-helpers/pull/18/commits/8715774b10637f1aacb45aaf03dfedb46af49910